code/stack/stack.py

```
@@ -264,4 +268,364 @@ def test():
+    print(D)
+
+    swapped_D = swap_indices_stack(D, S, 3, 4)
+    print(swapped_D)
+
+
+def test_deque():
+
+    D = ArrayDeque()
+
+    D.add_last(5) 
+    D.add_first(3) # – [3, 5]
```

Ruff (0.14.0)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)